code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -102,6 +85,7 @@
 	var/health_cost = 0
 	/// Have we already been positioned into our starting location?
 	var/positioned = FALSE
+	default_button_position = SCRN_OBJ_CULT_LIST
```

Comment:
this needs to go above the variables defined on it's type
Type Definition -> Inherited Vars -> Newley Defined Vars